### PR TITLE
New classic stylesheet

### PR DESCRIPTION
**Pull request by defnax**

### CODE REVIEW

File: retroshare-gui/src/qss/retroclassic.qss [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+
+[valid=false] {
+	background: rgba(255,0,0,15%);
+}
+
+[WrongValue="true"] {
+	background-color: #FF8080;
+}
+
+/* HeaderFrame & TitleBarFrame */
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss:422]: Move all this block before line 38 in Standard rules.

File: retroshare-gui/src/qss/retroclassic.qss [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+	color: white;
+}
+
+QFrame[objectName^="toolBarFrame"],
+QFrame[objectName^="toolBarFrameTop"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
+
+ChatWidget QFrame#titleBarFrame {
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss]: This use generic "titleBarFrame" name. If not generic, rename it.

File: retroshare-gui/src/qss/retroclassic.qss [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+/**** Standard rules ****/
+
+/* Transparent background objects = name starting by trans_*/
+QObject[objectName^="trans_"] {
+	background: transparent;
+}
+/* Specific for scrollarea contents */
+QScrollArea[objectName^="trans_"] > QWidget > QWidget {
+	background: transparent;
+}
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss:22]: Why rewrite this? It's already the same in default.qss

File: retroshare-gui/src/qss/retroclassic.qss [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+
+ChatWidget QFrame#titleBarFrame {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
+
+ChatWidget QFrame#titleBarFrame > *:!hover {
+	background: transparent;
+	color: black;
+}
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss]: Move all this block before line 38 in Standard rules.

**Comment by PhenomRetroShare** [comment]
Why rewrite this? It's already the same in default.qss
Only write what you want to be changed.

First, merge PR #2510...

File: retroshare-gui/src/qss/retroclassic.qss (hunk 2 above)
Comment (defnax) [reply retroshare-gui/src/qss/retroclassic.qss]: else everything get the same color, i wanted only for the chatwidget titleBarFrame the color

**Comment by defnax** [comment]
> Why rewrite this? It's already the same in default.qss Only write what you want to be changed.
> 
> First, merge PR #2510...

as you said the default.qss is for a blank empty stylesheet, you dont wanted the the classic colors( it breaks the colors in dark system styles on linux) in default.qss, then i generated a own one classic stylesheet retroclassic.qss to look similar like from release 0.6.6
in default.qss you doesnt added back the colors.

File: retroshare-gui/src/qss/retroclassic.qss (hunk 2 above)
Comment (PhenomRetroShare) [reply retroshare-gui/src/qss/retroclassic.qss]: So you have to rename it. Don't forget to add it to dark and light too.

**Comment by PhenomRetroShare** [comment]
In all case default is loaded. When selected, only it is loaded.
That's why I remove your white widget in default.

**Comment by defnax** [comment]
this is ready can you check @PhenomRetroShare

File: retroshare-gui/src/qss/retroclassic.qss [hunk 5]
```diff
@@ -0,0 +1,452 @@
...
+QFrame[objectName^="headerBFrame"] > *:!hover{
+	background: transparent;
+	color: white;
+}
+
+QFrame[objectName^="toolBarFrame"],
+QFrame[objectName^="toolBarFrameTop"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss:58]: Only these lines are needed as all other are in default.qss.

File: retroshare-gui/src/qss/retroclassic.qss [hunk 6]
```diff
@@ -0,0 +1,452 @@
...
+
+ChatWidget QFrame#titleBarFrame {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
+
+ChatWidget QFrame#titleBarFrame > *:!hover {
+	background: transparent;
+	color: black;
+}
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/qss/retroclassic.qss:68]: No ChatWidget::titleBarFrame, it was renamed headerBFrame to do what you've ask.

File: retroshare-gui/src/qss/retroclassic.qss (hunk 5 above)
Comment (defnax) [reply retroshare-gui/src/qss/retroclassic.qss:58]: i doesnt understand you, how will the rest work ?
rs will load then two qss at the same time?
on default.qss are importend stylesheets which is needed for the retroclassic.qss

Comment (PhenomRetroShare) [reply retroshare-gui/src/qss/retroclassic.qss:58]: As I've already wrote, default.qss is load every time (that's its name). When you choose another in list, RS loads it after default.qss. So you can override some properties.
The code is here:
https://github.com/RetroShare/RetroShare/blob/master/retroshare-gui/src/rshare.cpp#L690

Comment (defnax) [reply retroshare-gui/src/qss/retroclassic.qss:58]: ok i now i understand it, i need do some tests

**Pull request closed without merging**